include/conf.hpp

```diff
@@ -29,11 +31,16 @@ class Config {
     uint8_t icmp;
     uint32_t payload;
 
+    /* Link-layer types */
+    uint8_t wireless;
+    uint8_t wired;
+    void set_link_layer_type();
+
     /*  Output modification */
     uint8_t stats;
     uint8_t csv;
     uint8_t pcap;
-    uint8_t index;
+    int8_t index;
```

Comment:
This is fixed with #23, using `write_index`

Reply:
Resolved